cpp/ql/test/library-tests/CPP-205/elements.ql

```diff
@@ -1,6 +1,16 @@
 import cpp
 
+string describe(Element e) {
+  result = "from " + e.(Function).getFullSignature()
+  or
+  result = "from " + e.(FunctionDeclarationEntry).getFunction().getFullSignature()
+  or
+  result = "from " + e.(Parameter).getFunction().getFullSignature()
+  or
+  result = "from " + e.(ParameterDeclarationEntry).getFunctionDeclarationEntry().getFunction().getFullSignature()
```

Comment:
Seeing as we're enumerating these, we may as well make them even more descriptive, e.g. `"from parameter declaration entry for " + ...`